text/0000-fix-error.md

```diff
@@ -112,6 +136,14 @@ fn cause(&self) -> Option<&dyn Error> {
 This way, if an error defines `source`, someone calling the deprecated `cause`
 API will still get the correct cause type, even though they can't downcast it.
 
+## Stability
+
+The addition of `source` and the deprecation of `cause` will be instantly
+stabilized after implementing this RFC.
+
+The addition of the backtrace method and the entire backtrace API will be left
+unstable under the `backtrace` feature for now.
```

Comment:
This makes me much, much happier about the addition of backtrace information, thank you!